.github/workflows/wordlist.txt

```diff
@@ -1,1213 +1,3 @@
-SolverSettings
-SpaceOrganismTemplate
-spectrometric
-Spendiff
-Springer
-SQRT
-src
-SRND
-ssd
-Stahlhofen
-StandardOntogeny
-StartTime
-Stass
-StateVariable
-stateVariableParam
-Steimer
-stepwise
-stoichiometric
-stoichiometry
-Stotal
-Strougo
-Studien
-subcompartment
-subcompartments
-subcontainer
-subcontainers
-subfolder
-subfolders
-submenu
-submodel
-subpopulations
-subtree
-subviews
-Sufentanil
-Suillerot
-SumMetabolites
-superfamilies
-supportfiles
-Surg
-Swiatek
-Syst
-TableFormula
-tableParam
-TableParameter
-tabu
-Tanaka
-TANH
-TargetSimulations
-targetUnit
-tcolorbox
-tD
-tDEnd
-TDI
-tDj
-tDlast
-Telle
-tEnd
-tEndAUC
-tEndThalfAUC
-Terenji
-textcomp
-th
-Thalf
-Theil
-Thelen
-Theophylline
-Theor
-Ther
-thetable
-threeparttablex
-ths
-tikz
-TimeProfile
-TimeUnit
-Tingleff
-Tlag
-tlast
-toBaseUnit
-TOC
-toDisplayUnit
-tooltip
-totalSensitivityThreshold
-toUnit
-Toxicol
-Tozer
-transcapillary
-transcellular
-transcriptional
-TRANSIL
-translational
-TSmax
-tstart
-Tsutsumi
-txt
-typora
-Tytgat
-uc
-UCRL
-Ueffing
-UGT
-UI
-uk
-un
-Uncheck
-Uncompetitive
-und
-undissociated
-Undock
-ungroup
-UniGene
-unlink
-unlinked
-Unserer
-unstirred
-unticking
-untransformed
-Urooj
-UseDerivedValues
-UseJacobian
-userdefinedsnippets
-USERID
-Vakalopoulos
-valueAt
-ValuePoint
-ValuePointAlreadyExistsForPointException
-variabilities
-variated
-vasc
-vascularization
-VCH
-Vd
-VenousBlood
-Vergin
-Verlag
-Vis
-visualstudio
-vitro
-vitrosystem
-vivo
-Vmax
-Voith
-volumeParams
-Vossen
-VSCode
-Vss
-Watelet
-WB
-Weibull
-weightMax
-weightMin
-weightUnit
-Weinheim
-Wendl
-wikipedia
-Willmann
-withcellular
-workstep
-Wurthwein
-www
-xcolor
-XDimension
-Xenobiotica
-xenografts
-xls
-xlsx
-xml
-xtol
-xValues
-XYZ
-Yaguti
-Yakugaku
-Yamamoto
-Yaroslavsky
-Yassen
-Yeo
-Yeung
-Yoshitsugu
-yout
-yValues
-Zasshi
-Zeit
-Zhu
-Zischka
-zonated
-zonation
-Zwitterions
-OSPSuite
-csv
-mucosal
-rhsFormula
-rhs
-simulationResults
-outputSchema
-endTime
-outputSchema
-startTime
-populationCharacteristics
-pkParameter
-PKAnalyses
-outputSelections
-SAResult
-pharmacokinetic
-multiscale
-Anwendung
-nhanes
-ncbi
-unigene
-de
-itraconazole
-Emax
-pharmacodynamics
-bioavailability
-biliary
-glomerular
-lipophilicity
-endosome
-Ki
-NonMem
-userid
-jacobian
-uncheck
-Ctrl
-QualificationRunner
-educt
-biometrics
-tDi
-dDi
-PKSim
-hematocrit
-pka
-periportal
-uncompetitive
-basolat
-bbb
-brn
-intracellularly
-IVIVE
-localizations
-Localizations
-luminal
-vasculature
-vmax
-pls
-vasend
-bc
-endo
-PortalVein
-BloodCells
-DV
-DVID
-DVNAME
-DVUNIT
-nmdat
-NaN
-WIP
-vas
-closable
-basolatateral
-runAsync
-simulationRunArgs
-runtime
-LLOQs
-Lloq
-closeable
-Anwendungen
-Claassen
-CMax
-CPT
-doi
-Ince
-Jia
-Jonsson
-Mavroudis
-PBTK
-Pharmacometrics
-PLoS
-Preuss
-psp
-Sjögren
-Teutonico
-toxicokinetics
-Validator
-Workgroup
-Curr
-Des
-EFPIA
-jcph
-JF
-Tarning
-TG
-preselect
-Arterialized
-GLP
-ables
-igures
-istings
-tlf
-reportingengine
-parameteridentification
+PKParameter
+PKSim
+userid
```

Comment:
Spellchecker is now using 2 word lists:
* global word list under https://github.com/Open-Systems-Pharmacology/Workflows/blob/main/Data/wordlist_osp_global.txt 
* local word list (this one)

The idea is: words which are **always** valid, should be added to the global word list.
Words which sre valid only **in the context of the current repo**, should be added to the local word list.
E.g. **PKSim** is in general not valid (it should be **PK-Sim**), but in this repo it is used e.g. in "PKSim.CLI" etc. - thus adding it to the local word list.

Reply:
>Not sure about a global wordlist....should we have a way to add some words at the repo level?

Yep, that's exactly what is described above. Global (= always valid) + local (valid for this repo only)